docs/Community/Repositories.md

```diff
@@ -0,0 +1,122 @@
+organization.  This makes it clear that Open Enclave itself, rather than
+a just a random developer, depends on the fork.
+
+#### Rules
+
+   1. Must adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
+      statement in their repo.
+   2. Must use the same license and copyright rules as the forked project.
+   3. Must adopt the [DCO](Contributing.md#developer-certificate-of-origin)
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Repository must be approved by [SIG-Architecture](sig-architecture/README.md).
+
+Other than rule 2, all other rules are the same as for core repositories.
+
+## Creating Repositories
+
+Requests for creating repositories can be made by filing an
+[Open Enclave issue](https://github.com/openenclave/openenclave/issues)
+if one does not already exist.
+
+In addition to stating any deviations from the Open Enclave defaults
+discussed under Rules above, any request to create a new repository
+should also answer the following questions:
+
+1. Should github issues be filed in the new repo, or in the main
+   openenclave repo?
+2. Should issues in the new repo be discussed in the master openenclave
+   triage meeting, or in some separate process?
+3. Should one or more of the existing SIGs (e.g.,
+   [SIG-Architecture](sig-architecture/README.md),
+   [SIG-Testing](sig-testing/README.md)) oversee relevant aspects of the
+   new repo, or not?
+4. Will the repo be public or private, and if private, why?
+
+Once a repository is created, answers to 1-3 are also expected to be
+explained in the repository's README.md file, or other files reachable
+from that file.
+
+## Removing Repositories
+
+As important as it is to add new repositories, it is equally important to prune
+old repositories that are no longer relevant or useful.
+
+It is in the best interests of everyone involved in the Open Enclave community
+that our various projects and repositories are active and healthy. This ensures
+that repositories are kept up to date with the latest Open Enclave wide
+processes, it ensures a rapid response to potential required fixes (e.g.,
+critical security problems), and (most importantly) it ensures that
+contributors and users receive quick feedback on their issues and
+contributions.
+
+### Grounds for removal
+
+Core repositories may be removed from the project if they
+are deemed _inactive_. Inactive repositories are those that meet any of the
+following criteria:
+
+   * There are no longer any active maintainers for the project and no
+     replacements can be found.
+   * All PRs or Issues have gone un-addressed for longer than six months.
+   * There have been no new commits or other changes in more than a year.
```

Comment:
Is it unreasonable to allow for some repos that may not need to churn on a regular basis (e.g. samples against stable OE SDK APIs)?

Reply:
This bullet was copied from the Kubernetes one, but I'll remove it, since I agree with the point and line 120 already covers the case where no changes have been done when some are actually needed.